_resources/participatory-design-inclusivity-heuristics.md

```diff
@@ -0,0 +1,248 @@
+      <th scope="col" class="col-indicator">Indicator</th>
+      <th scope="col" class="col-why-it-matters">Why it matters</th>
+      <th scope="col" class="col-good-signs">Good signs</th>
+      <th scope="col" class="col-warning-signs">Warning signs</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th colspan="4" id="account-for-context"><p>Account for context</p></th>
+    </tr>
+    <tr>
+      <th scope="row" class="col-indicator"><p>Values</p></th>
+      <td class="col-why-it-matters"><p>Values tie people together in ways that are difficult to break, especially when they’re not made explicit. Accounting for values helps teams approach design honestly and respectfully.</p></td>
+      <td class="col-good-signs"><p>The team identifies and reflects on its values. The team respects the values of the people affected by its work. The team acknowledges when it seeks to challenge the values of others.</p></td>
+      <td class="col-warning-signs"><p>The team doesn’t identify or discuss its values. The team does not see or consider the merits of the values of others. The team misleads others when attempting to change their values.</p></td>
+    </tr>
+    <tr>
+      <th scope="row" class="col-indicator"><p>Power dynamics</p></th>
+      <td class="col-why-it-matters"><p>Power dynamics shape all interpersonal relationships, as well as our collective ability to imagine potential futures. Power dynamics affect what’s considered possible, and the theories of change we will knowingly pursue.</p></td>
+      <td class="col-good-signs"><p>Power is critically assessed. Team members understand who makes decisions and how (see “decision-making processes,” below). Team members in positions of power seldom exercise their power to limit debate (in other words, it practices forbearance). Team members in positions of lower power are able to hold their superiors to account without fear of reprisal. The team actively reduces power imbalances whenever possible; for example, by prioritizing product features for people who need them most (not necessarily building for the majority).</p></td>
+      <td class="col-warning-signs"><p>Power is seldom discussed. Team members are unclear who makes decisions and how (see “decision-making processes,” below). Team members in positions of power often exercise their power to limit debate. Team members in positions of lower power remain silent due to “professionalism,” differences in status (for example, refraining from speaking for fear of humiliating someone with higher status), or fear of reprisal. The team exercises power without considering the people implicated in its decisions.</p></td>
+    </tr>
+    <tr>
+      <th scope="row" class="col-indicator"><p>Narratives</p></th>
+      <td class="col-why-it-matters"><p>Narratives shape how we make sense of the world. Accounting for narratives helps teams see the difference between what is real and what is true, which encourages more diverse, accurate design.</p></td>
```

Comment:
Consider using a simple word (`stories`) instead of a complex one (`narratives`). What does it mean to `account for` something? How do you want readers to understand "the difference between what is real and what is true"? What is `diverse design`? What is `accurate design`? How would you know if your design is diverse or accurate?

Reply:
>What does it mean to account for something? 

Account for = Pay attention to the ways in which the team is (1) making sense of things or (2) prescribing roles or anticipating the behaviors of others. Stereotyping someone is, to some extent, robbing them of the agency in owning their own narrative.

Reply:
>What is diverse design? What is accurate design? How would you know if your design is diverse or accurate?

Warning: this sound more highfalutin than I want it to at 12:13am... 

Diverse design is design without orthodoxy. It's design that accounts for and allows for multiple perspectives and multiple uses. Accurate design is design that accurately reflects the complexity and messiness of real people.

You'll know a design is diverse and accurate if it accounts for the needs of both [extremes and mainstreams](http://www.designkit.org/methods/45)... I guess in usability tests and stakeholder interviews :)?